test/xpu/fin_grain/test_ops_xpu.py

```diff
@@ -149,10 +150,12 @@ class TestCommon(TestCase):
     #@ops(_xpu_computation_ops_no_numpy_ref, dtypes=any_common_cpu_xpu_all)
     @ops(_xpu_computation_ops, dtypes=cpu_xpu_all)
     def test_compare_cpu(self, device, dtype, op):
-        self.proxy = Namespace.TestCommonProxy()
-
-        test_common_test_fn = get_wrapped_fn(Namespace.TestCommonProxy.test_compare_cpu)
-        test_common_test_fn(self.proxy, device, dtype, op)
+        if dtype in op.supported_dtypes(device):
```

Comment:
We have not supported supported_dtypes for XPU device. What is the return of the API? dtypes are claimed by CPU device I remember?

Reply:
Resolved. It's op dtype. May not align CUDA. But it is reasonable to the operator tested. Will have further enhancement in test infrastructure to support XPU specific op dtype.